serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -194,15 +203,15 @@ After building your Serveless Workflow image using Quarkus CLI, you can test the
 .Example JVM image
```

Comment:
@spolti Can we change the JVM and Native examples to switch tabs?

Reply:
sure.